specification/columns/capacityreservationstatus.md

```diff
@@ -0,0 +1,42 @@
+# Capacity Reservation Status
+
+Capacity Reservation Status indicates whether the charge represents either the consumption of the [*capacity reservation*](#glossary:capacity-reservation) identified in the CapacityReservationId column or when the *capacity reservation* is unused.
+
+The CapacityReservationStatus column adheres to the following requirements:
+ * CapacityReservationStatus MUST be present in the billing data when the provider supports *capacity reservations* and MUST be of type String.
+ * CapacityReservationStatus MUST be null when CapacityReservationId is null and MUST NOT be null when CapacityReservationId is not null.
```

Comment:
SHOULD NOT be null when CapacityReservationId is not null and the charge is related to a Used or Unused capacity reservation.